src/types.ts

```diff
@@ -1,5 +1,7 @@
 import { z } from 'zod';
 
+export const SUPPORTS = ['API3Market', 'ChainAPI', 'dAPIs', 'OEVRelay'];
```

Comment:
I would either do camel case (Api3Market, ChainApi, Dapis, OevRelay) or full names (API3 Market, ChainAPI, dAPIs, OEV Relay)

Reply:
I made these all fully lower case for now to avoid any kind of ambiguity or confusion. It seems the simplest to me (e.g. the example above is pascal case, not camel case 😬). What do you think?